resources/qml/CameraPositionPanel.qml

```diff
@@ -104,7 +104,7 @@ UM.Dialog
             
             text: "zoom"
             value: manager.zoom
-            validator: DoubleValidator {bottom: -10; top: 10;}
+            validator: DoubleValidator {bottom: -0.495; top: 10;}
```

Comment:
unnamed but especially "Weird" hard-coded values benefit from a comment line

Reply:
The value is obtained because it is hard coded in the the Uranium CameraTool.py line [318](https://github.com/Ultimaker/Uranium/blob/c68020a0f8eb974e7ccd67c8430e9ed1acec981a/plugins/Tools/CameraTool/CameraTool.py#L318). 

Which is presumably set to guard the limit when for the function _zoomCamera when zoomfactor goes to -0.5. At that point a division bij zero occurs and the function domain below  -0.5 is undefined.